main.cpp

```diff
@@ -78,6 +126,28 @@ You are going to write 10 UDTs in project3.
     2) Delete Song Data
     3) Replace Song Data
  */
+struct Song 
+{
+    //song name
+    bool hasSongName = true;
+    //artist name
+    std::string artistName = "The Artist";
+    //genre
+    std::string genre = "Rock";
+    //song description
+    bool hasSongDescription = false;
+    //song length
+    float songLength = 3.13f; //minutes
+
+    //provide access to song
+    void accessSongData ( Song mySong );
+    //delete song data
+    void deleteSongData ( Song mySong );
+    //replace song data
+    void replaceSongData ( Song mySong ); // or would it be better to have different parameters here like ( Song replacedSongData )?
```

Comment:
All of these member functions imply that you'll be passing a `Song` **TO** this instance of `Song`:
```
Song aSong;
Song someOtherSong;

aSong.replaceSongData( someOtherSong );
```
for example, you might be copying the properties of `songOtherSong` into `aSong`.
The purpose of `accessSongData` is not immediately obvious the way it is written. 
What is `deleteSongData()` supposed to do?  Which instance of `Song` is having their data deleted? `the one being passed in? or the one who is having `deleteSongData()` called upon it? 
```
aSong.deleteSongData( someOtherSong ); //Which object is having their data deleted?
``` 
if `someOtherSong` is having its data deleted, Why is `aSong` the one doing the deletion? 
- [ ] revise.